tests/test_dataelem.py

```diff
@@ -472,7 +472,7 @@ def check_empty_text_element(value):
         }
         config.use_none_as_empty_text_VR_value = use_none
         ds = Dataset()
-        ds.is_little_endian = True
+        ds._is_little_endian = True
```

Comment:
A lot of these explicit settings of `_is_little_endian` and `_is_implicit_VR` in the tests get rolled in the `save_as()`  changes in Stage 2